Source/WTF/wtf/cocoa/SoftLinking.h

```diff
@@ -28,6 +28,9 @@
 #import <objc/runtime.h>
 #import <wtf/Assertions.h>
 
+#define SOFT_LINK_DLSYM_SECTION "__dlsym_cstr"
+#define SOFT_LINK_GETCLASS_SECTION "__getClass_cstr"
```

Comment:
Is there a reason to not include `__TEXT,` in these? Or even the whole `__attribute__`.

Reply:
Thank you! No reason, and perhaps putting the `__attribute__` here improves readability.